firebase-appdistribution/src/main/res/layout/activity_feedback.xml

```diff
@@ -4,6 +4,7 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:padding="24dp"
```

Comment:
Is this just generated XML changes?  I kind of just skimmed this.

Reply:
These add the info text to our proof-of-concept UX.